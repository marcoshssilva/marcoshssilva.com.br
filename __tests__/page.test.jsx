import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Home from '@/app/page';

describe('Home', () => {
  it('render component', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
