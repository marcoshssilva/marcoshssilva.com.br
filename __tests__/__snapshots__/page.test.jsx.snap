// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render component 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-1a04xh8-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      >
        <a
          class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
          href="/"
        >
          marcoshssilva
        </a>
        <a
          class="MuiTypography-root MuiTypography-h6 css-oblp1g-MuiTypography-root"
          href="/"
        >
          .com.br
        </a>
        <div
          class="MuiBox-root css-i9gxme"
        />
        <div
          class="MuiBox-root css-1guk29"
        />
      </div>
    </div>
  </header>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg min-h-screen pt-[72px] lg:pt-0 css-1q93duj-MuiContainer-root"
  >
    <div
      class="grid absolute md:bottom-1/2 md:translate-y-1/2 px-6 grid-cols-1 md:grid-cols-12 gap-5 MuiBox-root css-0"
    >
      <div
        class="col-span-1 md:col-span-7 h-full MuiBox-root css-0"
      >
        <div
          class="flex flex-col MuiBox-root css-0"
        >
          <h1
            class="MuiTypography-root MuiTypography-h1 css-611upe-MuiTypography-root"
          >
            Olá, eu sou
          </h1>
          <h1
            class="MuiTypography-root MuiTypography-h1 css-mh4pck-MuiTypography-root"
          >
            Marcos Henrique.
          </h1>
          <h1
            class="MuiTypography-root MuiTypography-h1 css-vm9qd0-MuiTypography-root"
          >
            Desenvolvo aplicações para a web
          </h1>
          <p
            class="MuiTypography-root MuiTypography-body1 css-150xbqw-MuiTypography-root"
          >
            Sou um Desenvolvedor FullStack com alguns anos de experiência.
            <br />
            Atualmente, focado em desenvolvimento de aplicações voltadas para microserviços e escalabilidade vertical.
          </p>
        </div>
        <div
          class="pt-6 pb-3 MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Siga-me pelas redes sociais:
          </p>
        </div>
        <div
          class="flex gap-3 MuiBox-root css-0"
        >
          <a
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge css-9y1egq-MuiButtonBase-root-MuiButton-root"
            href="https://github.com/marcoshssilva"
            tabindex="0"
            target="_blank"
          >
            <span
              class="MuiButton-startIcon MuiButton-iconSizeLarge css-htszrh-MuiButton-startIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="GitHubIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
                />
              </svg>
            </span>
            GitHub
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </a>
          <a
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge css-9y1egq-MuiButtonBase-root-MuiButton-root"
            href="https://www.linkedin.com/in/marcos-henrique-santana/"
            tabindex="0"
            target="_blank"
          >
            <span
              class="MuiButton-startIcon MuiButton-iconSizeLarge css-htszrh-MuiButton-startIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="LinkedInIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                />
              </svg>
            </span>
            LinkedIn
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </a>
        </div>
      </div>
      <div
        class="col-span-1 md:col-span-5 h-full MuiBox-root css-0"
      >
        <div
          class="h-full flex flex-col items-baseline w-full MuiBox-root css-0"
        >
          <form
            autocomplete="off"
            class="m-0 w-full py-6"
            novalidate=""
          >
            <span
              class="MuiTypography-root MuiTypography-caption mb-6 css-1sn4lm3-MuiTypography-root"
            >
              Gostaria de entrar em contato comigo?
            </span>
            <div
              class="MuiBox-root css-o9hh3a"
            />
            <div
              class="grid grid-cols-2 gap-2 mb-6 MuiBox-root css-0"
            >
              <div
                class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-filled MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-filled css-1mxeotk-MuiFormLabel-root-MuiInputLabel-root"
                  data-shrink="false"
                  for="nome"
                  id="nome-label"
                >
                  Seu Nome
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-125iqkp-MuiFormLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-1906lwe-MuiInputBase-root-MuiFilledInput-root"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiFilledInput-input css-1rv476z-MuiInputBase-input-MuiFilledInput-input"
                    id="nome"
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-filled MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-filled css-1mxeotk-MuiFormLabel-root-MuiInputLabel-root"
                  data-shrink="false"
                  for="email"
                  id="email-label"
                >
                  Seu Email
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-125iqkp-MuiFormLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-1906lwe-MuiInputBase-root-MuiFilledInput-root"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiFilledInput-input css-1rv476z-MuiInputBase-input-MuiFilledInput-input"
                    id="email"
                    required=""
                    type="email"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              class="grid grid-cols-1 gap-2 mb-6 MuiBox-root css-0"
            >
              <div
                class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-filled MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-filled css-1mxeotk-MuiFormLabel-root-MuiInputLabel-root"
                  data-shrink="false"
                  for="mensagem"
                  id="mensagem-label"
                >
                  Mensagem
                  <span
                    aria-hidden="true"
                    class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-125iqkp-MuiFormLabel-asterisk"
                  >
                     
                    *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-multiline css-13ietz0-MuiInputBase-root-MuiFilledInput-root"
                >
                  <textarea
                    aria-invalid="false"
                    class="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputMultiline css-sfsrwa-MuiInputBase-input-MuiFilledInput-input"
                    id="mensagem"
                    required=""
                    rows="5"
                    style="height: 0px; overflow: hidden;"
                  />
                  <textarea
                    aria-hidden="true"
                    class="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputMultiline css-sfsrwa-MuiInputBase-input-MuiFilledInput-input"
                    readonly=""
                    style="visibility: hidden; position: absolute; overflow: hidden; height: 0px; top: 0px; left: 0px; transform: translateZ(0); padding-top: 0px; padding-bottom: 0px; width: 100%;"
                    tabindex="-1"
                  />
                </div>
              </div>
            </div>
            <div
              class="flex justify-end MuiBox-root css-0"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-jcwuwq-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="submit"
              >
                <span
                  class="MuiButton-startIcon MuiButton-iconSizeMedium css-1d6wzja-MuiButton-startIcon"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="SendIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M2.01 21 23 12 2.01 3 2 10l15 2-15 2z"
                    />
                  </svg>
                </span>
                Envie-me uma mensagem
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</div>
`;
